src/sentry/integrations/jira_server/integration.py

```diff
@@ -223,11 +243,33 @@ def dispatch(self, request: Request, pipeline) -> Response:
             return pipeline.error("Could not fetch an access token from Jira")
 
 
-class JiraServerIntegration(JiraIntegration):
```

Comment:
This whole file is pretty much just copying Jira Cloud's `integration.py` methods that weren't already in here, and no longer subclassing from Jira Cloud.